e2e/jail/jail_test.go

```diff
@@ -154,10 +176,69 @@ func (s *JailTestSuite) TestEventSignal() {
 	response, err := responseValue.ToString()
 	s.NoError(err, "cannot parse result")
 
-	expectedResponse := `{"jsonrpc":"2.0","result":true}`
+	expectedResponse := `{"result":true}`
```

Comment:
This is just a consequence of removing unnecessary `"jsonrpc":"2.0"` from the response. You commented that we can implement this accordingly to `web3` spec so it will change once again.

Reply:
Ok, but confirm with Roman first just in case.

Reply:
It's confirmed. We can return value without encapsulating it in `{"result": value}`.